clang/include/clang/AST/StmtOpenMP.h

```diff
@@ -5779,6 +5779,80 @@ class OMPReverseDirective final : public OMPLoopTransformationDirective {
   }
 };
 
+/// Represents the '#pragma omp interchange' loop transformation directive.
+///
+/// \code{c}
+///   #pragma omp interchange
+///   for (int i = 0; i < m; ++i)
+///     for (int j = 0; j < n; ++j)
+///       ..
+/// \endcode
+class OMPInterchangeDirective final : public OMPLoopTransformationDirective {
```

Comment:
If the directive itslef can be used without clauses, need to introduce the directive at first and then add a clause in a separate patch

Reply:
Interchange construct extracted out into #93022